tests/test_fixture_start_end_test.py

```diff
@@ -10,6 +11,12 @@ def test_fixture_start_end_test(suite, suite_test, scope):
     test_event = suite_test.add_event()
 
     suite_test.depend_on_fixture(fixture)
+
+    if error_adder == 'add_error':
+        suite_test.when_run.error()
+    elif error_adder == 'add_failure':
+        suite_test.when_run.fail()
+
```

Comment:
Add:
```
...
else:
    assert error_adder is None
```
This is to avoid someone accidentally adding another flavor in the future, not changing the test accordingly